### PR TITLE
Bugfix/remove destructuring

### PR DESCRIPTION
**Pull request by elektronik2k5**
Fixes https://github.com/camacho/yarn-or-npm/issues/1

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-var yarnOrNpm = require('../index');
-
+const yarnOrNpm = require('../index');
 // Execute the command
 try {
-  const { status } = yarnOrNpm.spawn.sync(
-    process.argv.slice(2), { stdio: 'inherit' }
-  );
+  const status = yarnOrNpm.spawn.sync(
```
Comment (camacho) [inline bin/index.js]: will `const` throw in older versions of node?

Comment (elektronik2k5) [inline bin/index.js]: No, looks like it won't: http://node.green/#ES2015-bindings-const-basic-support

**Comment by camacho** [comment]
merged and published via npm! thank you for the contribution!

**Comment by elektronik2k5** [comment]
Yay, thank you! :)

**Pull request merged**